src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,20 @@
+// Libraries
 import React from 'react';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+// Styles
+import './UserInfo.scss';
+
+type Props = {
+  userName: string | undefined;
+  userEmail: string | undefined;
```

Comment:
Instead of using 'string | undefined', use 'string?' to mark optional properties in the Props type.

Reply:
btw, is this prop really could be undefined?, and will your app not break if we pass undefined?